### PR TITLE
feat: translation strings in footer

### PR DESCRIPTION
**Pull request by danielgent**
This is actually a working solution for how to do translations in our React components :sunglasses: 

#### TO NOTE
I've manually copied over these translation strings from `login2`.  We could also pull the login2 translation strings and use them in the React components...
*THOUGHTS?*  

https://github.com/sesamyab/auth/pull/594 I've gone ahead and pulled down the login2 translations on the next PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "dotenv": "16.4.5",
     "eslint": "8.57.0",
     "husky": "9.0.11",
+    "i18next": "^23.10.1",
```
Comment (danielgent) [inline package.json:83]: nothing to do with next.js unluckily for them... or not!

Comment (danielgent) [reply package.json:83]: If it works for me running wrangler locally then this must work on cloudflare workers right?

File: src/locales/en/default.json
Comment (danielgent) [inline src/locales/en/default.json:1]: I hate i18nexus for changing the order so much :cry: 

Micke said he'd asked them to sort this out, he also gave ideas of scripts for us to manually do the sorting but I think they should change this.

File: src/utils/components/Layout.tsx [hunk 2]
```diff
@@ -19,6 +22,15 @@ const globalDocStyle = (vendorSettings: VendorSettings) => {
   `;
 };
 
+i18next.init({
+  // TODO - will this come from vendor settings?
+  lng: "en",
```
Comment (danielgent) [inline src/utils/components/Layout.tsx:27]: ah no, this will come from the tenant settings. I could do that now actually :thinking:

Comment (danielgent) [reply src/utils/components/Layout.tsx:27]: I wonder how we would do that though, can the i18next initialisation happen asynchronously? e.g. could I do that in the render... or in the routes before we even get here :thinking:  

I'll try this on another PR. I'm sure there'll be a way

File: src/locales/en/default.json [hunk 3]
```diff
@@ -1,13 +1,15 @@
 {
+  "codeEmailEnterCode": "Enter the code at {{vendorName}} to complete the login",
+  "codeEmailTitle": "Welcome to {{vendorName}}! {{code}} is the login code",
+  "codeValid30Mins": "The code is valid for 30 minutes",
+  "contact_support": "Need Help?",
   "contactUs": "Contact us",
   "copyright": "Copyright © 2023 SESAMY. All rights reserved.",
-  "supportInfo": "If you have questions or need assistance, you can contact our support team",
-  "codeEmailTitle": "Welcome to {{vendorName}}! {{code}} is the login code",
+  "copyright_sesamy": "©2023 Sesamy",
```
Comment (danielgent) [inline src/locales/en/default.json:8]: https://github.com/sesamyab/auth/pull/594 I've gone ahead and pulled down the login2 translations on the next PR

File: src/utils/components/Layout.tsx [hunk 4]
```diff
@@ -99,6 +111,7 @@ const Layout: FC<LayoutProps> = ({ title, children, vendorSettings }) => {
                 </div>
                 <div class="flex flex-1 flex-col">
                   {children}
+                  {/* // do this on the next PR! */}
```
Comment (danielgent) [inline src/utils/components/Layout.tsx:114]: as promised :smile:

Comment (danielgent) [reply src/utils/components/Layout.tsx:114]: https://github.com/sesamyab/auth/pull/594

File: src/utils/components/Layout.tsx (hunk 2 above)
Comment (danielgent) [reply src/utils/components/Layout.tsx:27]: https://github.com/sesamyab/auth/pull/595 POC here!

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**